backend/src/solver/ConnectionSolver.py

```diff
@@ -4,6 +4,7 @@
 from itertools import chain
 import pandas as pd
 from datetime import datetime
+from time import time as get_time
```

Comment:
let's use `import time` and then `time.time()`

Reply:
I don't know why but when I tried to do it this way, time got error:
AttributeError: type object 'datetime.time' has no attribute 'time'
So import of datetime.time is hidden somewhere, but I cant say where, thats why I renamed the method.